packages/lockfile-lint/__tests__/cli.test.js

```diff
@@ -333,4 +333,22 @@ describe('Validator managers:', () => {
       ]
     })
   })
+  it('Integrity manager should return errors for lock file with packages with sha1 integrity', () => {
+    const result = ValidateIntegrityManager({
+      path: '__tests__/fixtures/package-lock-sha1.json',
+      type: 'npm',
+      validatorValues: ['https://github.com/LN-Zap/bolt11#0492874ea9ced4ab49bf0f59a62368687f147247']
+    })
+
+    expect(result).toEqual({
+      type: 'error',
+      errors: [
+        {
+          message:
+            'detected invalid integrity hash type for package: typescript@4.8.3-4b39c20edf186cd85bb485386ba7d48590c3bf0c\n    expected: sha512\n    actual: sha1-1ZNEUixLxGSmWnMKxpUAf9tm3Yg=\n',
+          package: 'typescript@4.8.3-4b39c20edf186cd85bb485386ba7d48590c3bf0c'
+        }
+      ]
+    })
+  })
 })
```

Comment:
I think this test should sort out the code coverage. Still I don't see it in the CodeCov report

Reply:
@lirantal Can you rerun the Codecov tests to verify coverage?

Reply:
Yes, trigging now.

Reply:
Great on coverage!

Reply:
Also, I think I either handled or answered with a question all of the comments above